### PR TITLE
Fix problem with duplicated keys (name label)

### PR DESCRIPTION
**Pull request by pbochynski**
Enables installation without helm (using helm template): https://github.com/kyma-project/kyma/issues/10939

### CODE REVIEW

File: resources/eventing/charts/publisher-proxy/templates/_helpers.tpl [hunk 1]
```diff
@@ -58,14 +58,6 @@ Selector labels for peerauthentication
 app.kubernetes.io/name: {{ include "publisher-proxy.fullname" . }}
```
Comment (sayanh) [inline resources/eventing/charts/publisher-proxy/templates/_helpers.tpl:58]: Can you please remove `publisher-proxy.peerauth.selectorLabels` as it is redundant now?

Comment (pbochynski) [reply resources/eventing/charts/publisher-proxy/templates/_helpers.tpl:58]: I can't. It is still used here: https://github.com/kyma-project/kyma/blob/e386863bb4cd486cd323df6c6d01af10c4a87537/resources/eventing/charts/publisher-proxy/templates/peerauthentication.yaml#L9

Comment (pbochynski) [reply resources/eventing/charts/publisher-proxy/templates/_helpers.tpl:58]: I can't. It is still used here: https://github.com/kyma-project/kyma/blob/e386863bb4cd486cd323df6c6d01af10c4a87537/resources/eventing/charts/publisher-proxy/templates/peerauthentication.yaml#L9

Comment (pbochynski) [reply resources/eventing/charts/publisher-proxy/templates/_helpers.tpl:58]: I didn't want to do a bigger clean-up. Just remove duplicated keys to make the output YAML template parseable.

**Review by sayanh — APPROVED** [review]

**Pull request merged**